requirements/pi.txt

```diff
@@ -39,26 +37,28 @@ neon-phal-plugin-fan~=0.1
 neon-phal-plugin-switches~=0.0.4
 neon-phal-plugin-linear_led~=0.2,>=0.2.2
 ovos-phal-plugin-balena-wifi~=1.0.0
-neon-phal-plugin-gui-network-client~=0.0.3
+neon-phal-plugin-gui-network-client~=0.0.4
 ovos-phal-plugin-network-manager~=1.1
-ovos-phal-plugin-wifi-setup~=1.1,>=1.1.1
-ovos-phal-plugin-oauth~=0.0.2
+ovos-phal-plugin-wifi-setup==1.1.2a1
+ovos-phal-plugin-oauth==0.0.3a2
 ovos-phal-plugin-alsa~=0.0.3
-ovos-phal-plugin-system~=0.0.4
+ovos-phal-plugin-system~=0.0.4 #,>=0.0.5a1
 ovos-phal-plugin-connectivity-events~=0.0.3
-# ovos-phal-plugin-homeassistant~=0.0.3,>=0.0.4a2
-ovos-phal-plugin-homeassistant~=0.0.3
+ovos-phal-plugin-homeassistant~=0.0.3 #,>=0.0.4a5
```

Comment:
Any reason not to go with the 0.0.4 alpha track?

Reply:
Just an attempt to keep alphas out of the stable releases where possible